src/react-infinite.jsx

```diff
@@ -1,6 +1,8 @@
 /* @flow */
 
 var React = global.React || require('react');
+var PropTypes = require('prop-types');
```

Comment:
I think these requires also need to attempt to access the global variable first so that this can still be used by people who have projects based on global variables.